src/api/tests/usersRoute.spec.ts

```diff
@@ -0,0 +1,43 @@
+import request, { Request } from 'supertest';
+
+import { sendInviteEmail } from '../services/kcUsersService';
+import { mockParticipant, mockUser } from './queryMocks';
+import useTestServer, { api } from './utils';
+
+jest.mock('../keycloakAdminClient', () => ({
+  getKcAdminClient: jest.fn(() => ({})),
+}));
+jest.mock('../services/kcUsersService', () => ({
+  queryUsersByEmail: jest.fn(() => [{}]),
+  sendInviteEmail: jest.fn(),
+}));
+
+describe('Users API tests', () => {
+  const withToken = useTestServer();
+
+  describe('When a user re-invites another user', () => {
+    afterEach(() => {
+      jest.restoreAllMocks();
+    });
+
+    test('From the same participant, the invitation is sent', async () => {
+      mockParticipant();
+      mockUser([{}, {}]);
```

Comment:
Each mock is only returned once. The first one will be returned when it loads the user being invited, and the second one will be returned when the logged-in user is loaded to check permissions. It doesn't matter here if they both have the same ID, it's just making sure it can find the logged-in user with the same participant.